site/content/en/docs/Reference/API/patches.md

```diff
@@ -0,0 +1,280 @@
+---
+title: "patches"
+linkTitle: "patches"
+type: docs
+weight: 15
+description: >
+    Patch resources
+---
+
+[strategic merge]: /references/kustomize/glossary#patchstrategicmerge
+[JSON6902]: /references/kustomize/glossary#patchjson6902
```

Comment:
Just noting the broken links until the glossary page is updated.

Reply:
I think we should just make them refer to other references for now.

https://github.com/kubernetes-sigs/kustomize/pull/5313/commits/d30e4570311d80cbf8533a235211f543e49343da